src/applications/registry.json

```diff
@@ -184,24 +184,6 @@
       "includeBreadcrumbs": true
     }
   },
-  {
-    "appName": "28-8832 Vocational Rehab Chapter 36 form",
-    "entryName": "chapter36-vre",
-    "rootUrl": "/employment/vocational-rehab-and-employment/application/chapter36",
-    "template": {
-      "vagovprod": false,
-      "layout": "page-react.html"
-    }
-  },
-  {
-    "appName": "28-1900 Vocational Rehab Chapter 31 form",
-    "entryName": "chapter31-vre",
-    "rootUrl": "/employment/vocational-rehab-and-employment/application/chapter31",
-    "template": {
-      "vagovprod": false,
-      "layout": "page-react.html"
-    }
-  },
```

Comment:
These entries were old hold-overs for applications that were never deployed. New versions of both forms are in the vsa-eBenefits product roadmap for near future development.